client/common.go

```diff
@@ -172,10 +173,12 @@ func (j *JenkinsCore) Request(method, api string, headers map[string]string, pay
 		req      *http.Request
 		response *http.Response
 	)
-
 	if req, err = http.NewRequest(method, fmt.Sprintf("%s%s", j.URL, api), payload); err != nil {
 		return
 	}
+	if j.Language != "" {
```

Comment:
How does the user can control this behavior?

Reply:
People maybe use like `jcli job type --language en`. The command can put the language to jenkinsCore

Reply:
I think it's not necessary that just add a language for command `jcli job type`. If there're three other commands need this kind of flags, how do you deal with that situation?

Reply:
I don't know how to set the global mode at the same time and how I can receive this value in the method.